src/main/java/jupyter/AsDisplayData.java

```diff
@@ -0,0 +1,19 @@
+package jupyter;
+
+import java.util.Map;
+
+/**
+ * Convenience interface for classes whose display data is known in advance.
+ *
+ * Classes implementing this interface are displayed via the {@link AsDisplayData#display} method
+ * by default, without needing to register a {@link Displayer} in advance.
+ */
+public interface AsDisplayData {
```

Comment:
Should this also include a default noop method to set the requested mime types? I don't see a drawback to allowing it in the API.

Reply:
Fixed. Note that this makes jvm-repr now require Java 8 (which I'm fine for me AFAIC).

Reply:
Yeah, I'm fine with that too since Java 7 is EOL.